analyzing-shakespeare/lib/macbeth/rexml/base_parser.rb

```diff
@@ -0,0 +1,9 @@
+require 'rexml/document'
+require_relative '../base_request'
+
+class BaseParser < BaseRequest
```

Comment:
It really bad place to you inheritance because you not extend child class
https://medium.com/aviabird/ruby-composition-over-inheritance-3ff786ad9e5d
pls read about 
Inheritance vs Composition